### ISSUE TITLE
 virtualbox - set nics to use paravirtualization for higher performance

### ISSUE THREAD

**Issue by nixjdm** [terminal-labs/rambo#4]
add the config change to the config.vm.provider block something like the below code

```
#!ruby

# setup paravirtualized network 
provider.customize ['modifyvm', :id, '--nictype1', 'virtio'] end
```

https://bitbucket.org/terminal_labs/rambo/issues/10/virtualbox-set-nics-to-use

**Comment by verhulstm** [comment]
the new base images made with packer should all use network paravirtualization by default. We can also add the directive into the vagrant config file.

**Comment by nixjdm** [comment]
fixed with #16
